filebeat/module/nginx/_meta/kibana/6/dashboard/ml-nginx-remote-ip-url-explorer.json

```diff
@@ -325,11 +325,11 @@
                             "url": "https://basemap.nationalmap.gov/arcgis/services/USGSTopo/MapServer/WMSServer"
                         }
                     }, 
-                    "title": "ML Nginx Access Map", 
+                    "title": "ML Nginx Access Map 7.x", 
                     "type": "tile_map"
                 }
             }, 
-            "id": "ML-Nginx-Access-Map", 
+            "id": "ML-Nginx-Access-Map-7x-7x", 
```

Comment:
Good catch. I tried to prevent these double replaces. Need to check if it's just a left over and I need to migrate it again or if it is one of the edge cases.

Reply:
Was an edge case, hopefully fixed now.